build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -270,9 +274,30 @@ private static void RemovePdbsFromDir(string directory)
                 GenerateNuGetPackagesArchive(c, dotnet, sdkOutputDirectory);
             }
 
+            CopyMSBuildTargetsToSDKRoot(sdkOutputDirectory);
+
             return c.Success();
         }
 
+        private static void CopyMSBuildTargetsToSDKRoot(string sdkOutputDirectory)
+        {
+            var msbuildTargetsDirectory = Path.Combine(sdkOutputDirectory, "runtimes", "any", "native");
```

Comment:
Looking at the `Microsoft.Net.Compilers.netcore\1.3.0` package from Roslyn, the `csc.exe` and `vbc.exe` files come in `runtimes\any\native` as well.

Why wouldn't these binaries come via the "normal" way of `lib\<TFM>`?  That's how `NuGet.CommandLine.XPlat.dll` is packaged.